src/cmd/serve.rs

```diff
@@ -53,10 +53,6 @@ enum ChangeKind {
     Config,
 }
 
-// Uglified using uglifyjs
-// Also, commenting out the lines 330-340 (containing `e instanceof ProtocolError`) was needed
-// as it seems their build didn't work well and didn't include ProtocolError so it would error on
-// errors
```

Comment:
can you update the license at the top and indicate the version?

Reply:
Something like this (before `const LIVE_RELOAD`)?
```suggestion
// This is dist/livereload.min.js from the LiveReload.js v3.0.0 release
```

The text of the [LICENSE file in the LiveReload.js repository](https://github.com/livereload/livereload-js/blob/master/LICENSE) remains the same as what's currently in Zola.

Reply:
Yep that's fine!

Reply:
This is added in now.